lib/chrono_model/time_machine/timeline.rb

```diff
@@ -28,7 +28,7 @@ def timeline(record = nil, options = {})
             # `join` first, then use `where`s
             relation =
               if ass.belongs_to?
-                relation.joins("LEFT JOIN #{association_quoted_table_name} ON #{association_quoted_table_name}.#{ass.association_primary_key} = #{quoted_table_name}.#{ass.foreign_key}")
+                relation.joins("LEFT JOIN #{association_quoted_table_name} ON #{association_quoted_table_name}.#{ass.association_primary_key} = #{quoted_table_name}.#{ass.association_foreign_key}")
               else
                 relation.joins("LEFT JOIN #{association_quoted_table_name} ON #{association_quoted_table_name}.#{ass.foreign_key} = #{quoted_table_name}.#{primary_key}")
```

Comment:
other occurrence of `ass.foreign key`

Reply:
in this case we should keep the ass.foreign_key because we are in the `has_many` case.
foo has many bars
and we are in :bars association
association.foreign_key return foo_id
association.association_foreign_key returns :bar_id because it does "Bar" (i.e class_name).foreign key => bar_id

Reply:
Ok thanks